lib/Branding/index.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import logoSVG from '../../assets/logo.svg';
+
+/**
+ * @usage
+ *
+ * <Branding url="http://example.com/myimage.png" alt="Alt Tag" />
+ */
+
+const Branding = props => {
```

Comment:
`Branding` doesn't really say what this component outputs. It seems to output a logo so maybe `Logo`?

Reply:
Renamed